holoviews/plotting/plot.py

```diff
@@ -1423,7 +1423,7 @@ def _get_range_extents(self, element, ranges, range_type, xdim, ydim, zdim):
 
         return (x0, y0, x1, y1)
 
-    def get_extents(self, element, ranges, range_type='combined', dimension=None, xdim=None, ydim=None, zdim=None, **kwargs):
+    def get_extents(self, element, ranges, range_type='combined', dimension=None, xdim=None, ydim=None, zdim=None, lims_as_soft_ranges=False, **kwargs):
```

Comment:
hmm are you hinting at the `get_extents` method of the `class GenericOverlayPlot`?

It doesn't appear to be necessary because even for overlay plots, GenericElementPlot's version of `get_extents` with `lims_as_soft_ranges` set to True seems to be called anyway. But honestly it's all pretty confusing and get_extents gets called numerous times so I could be missing something.

Reply:
I'm not hinting at anything specific; just that other classes could also need to implement `lims_as_soft_ranges` if they don't have the `super`. But I think we should wait and see if there is a demand for it, before doing anything.